graphene-pore/calc_numberdensity.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+import mdtraj as md
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import os
+import sys
+from mtools.gromacs.gromacs import make_comtrj
```

Comment:
Could you just copy this function from the mtools source code to here as a function? This would allow us to dropo the dep. Maybe we could also move these into another place for cleanliness, like `utils` folder or something

Reply:
For now I just put `make_comtrj` into a `utils` directory within this example.  Can be moved further if needed.